src/bob/api/routes.clj

```diff
@@ -47,9 +47,9 @@
                         :- String]
           :body [pipeline Pipeline]
           :summary "Creates a new pipeline in a group with the specified name.
-                   Takes list of steps, the base docker image and a list of environment vars
-                   as POST body."
-          (p/create group name (:steps pipeline) (:vars pipeline) (:image pipeline)))
+                   Takes list of steps, the base docker image, a list of environment vars
+                   and a list of artifacts generated from pipeline as POST body."
+          (p/create group name (:steps pipeline) (:vars pipeline) (:artifacts pipeline) (:image pipeline)))
```

Comment:
can add a unit test for this in https://github.com/bob-cd/bob/blob/master/test/bob/pipeline/core_test.clj#L43